_locales/it/messages.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": {
+    "message": "Tabs aside",
+    "description": "name of the extension. Displayed in the manifest and pane header"
+  },
+  "description": {
+    "message": "Il classico menu di Microsoft Edge \"Tabs Aside\"",
+    "description": "Extension description"
+  },
+  "author": {
+    "message": "Michael \"XFox\" Gordeev",
+    "description": "Author name"
+  },
+  "options": {
+    "message": "Opzioni",
+    "description": "Alternative text for options button in the pane"
+  },
+  "closePanel": {
+    "message": "Chiudi",
+    "description": "Alternative text for close panel button"
+  },
+  "loadOnRestore": {
+    "message": "Riapri le schede al ripristino",
+    "description": "Label for option"
+  },
+  "showDeleteDialog": {
+    "message": "Mostra una finestra di conferma prima di eliminare qualcosa",
+    "description": "Label for option"
+  },
+  "swapIconAction": {
+    "message": "Sposta le finestre in Tabs Aside al click sull'estensione (%TOGGLE_SHORTCUT% o click con il tasto destro sull'icona dell'estensione per aprire il pannello)",
```

Comment:
Google Translate:
> Move windows in Tabs Aside upon click on extension (% TOGGLE_SHORTCUT% or right click on extension icon to open panel)

Expected translation:
> Move **tabs aside** upon click on extension **icon** (% TOGGLE_SHORTCUT% or right click on extension icon to open panel)

Notes: In English first part reads: "Set tabs aside". Here, "tabs aside" is not a name. "Set" is a verb, "tabs" is an object and "aside" is an attribute. So this statement is similar to "Put (browser) tabs away"

Reply:
When I translated that I thought that sounds better in Italian because tabs are "moved" into the extension.
I'll replace "Finestre" with "schede" because finestre means windows and schede means tabs.